Lexplorer.sln

```diff
@@ -7,7 +7,7 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Lexplorer", "Lexplorer\Lexp
 EndProject
 Project("{D954291E-2A0B-460D-934E-DC6B0785DB48}") = "Shared", "Shared\Shared.shproj", "{E3CCB3F9-47E9-4F8E-85D6-9C964F24367C}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "xUnitTests", "xUnitTests\xUnitTests.csproj", "{246057E4-2CD6-46C3-9A50-37D8E365DD1E}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "xUnitTests", "xUnitTests\xUnitTests.csproj", "{246057E4-2CD6-46C3-9A50-37D8E365DD1E}"
```

Comment:
Did that just get in by mistake or is the GUID change intentional?

Reply:
Not sure why this changed, I've seen this before also. Can be omitted

Reply:
This seems to be automatically fixed by VS. VS changed the project type from FAE04EC0-301F-11D3-BF4B-00C04F79EFBC - C#, to 9A19103F-16F7-4668-BE54-9A1E7A4F7556 - .net core.

May be because of different VS versions. When changing back to FAE04EC0-301F-11D3-BF4B-00C04F79EFBC it fails to build on my machine. Will leave this change in.

Reply:
Ah okay, I didn't realise you would still be looking further at this - I already merged with the change.

Not sure how this started of as .Net core - I initially started working in a Windows VM with Visual Studio Code. I think this is where I created this project. In the meantime I'm on Visual Studio for Mac Preview, so maybe the fact that we're basically using 3 different IDEs here is the culprit. I never had it fail to build though.